docker/minder/Dockerfile

```diff
@@ -18,11 +18,11 @@ ENV APP_ROOT=/opt/app-root
 ENV GOPATH=$APP_ROOT
 
 WORKDIR $APP_ROOT/src/
-ADD go.mod go.sum $APP_ROOT/src/
+ADD ../../go.mod ../../go.sum $APP_ROOT/src/
```

Comment:
We're setting the context to the root of the git repo. Do we need this directory traversal? It shouldn't need it in theory.